MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -272,7 +272,8 @@ private bool IsHwTonemapAvailable(EncodingJobInfo state, EncodingOptions options
                 var isNvdecDecoder = vidDecoder.Contains("cuda", StringComparison.OrdinalIgnoreCase);
                 var isVaapiDecoder = vidDecoder.Contains("vaapi", StringComparison.OrdinalIgnoreCase);
                 var isD3d11vaDecoder = vidDecoder.Contains("d3d11va", StringComparison.OrdinalIgnoreCase);
-                return isSwDecoder || isNvdecDecoder || isVaapiDecoder || isD3d11vaDecoder;
+                var isVideoToolBoxDecoder = vidDecoder.Contains("videotoolbox", StringComparison.OrdinalIgnoreCase);
+                return isSwDecoder || isNvdecDecoder || isVaapiDecoder || isD3d11vaDecoder || isVideoToolBoxDecoder;
             }
```

Comment:
```
            if (string.Equals(state.VideoStream.Codec, "hevc", StringComparison.OrdinalIgnoreCase)
                && state.VideoStream.VideoRange == VideoRange.HDR
                && state.VideoStream.VideoRangeType == VideoRangeType.DOVI)
...
```

Does it really handle the DoVi Profile 5 dynamic metadata?

Reply:
I'm still modifying the implementation and verifying each format. Unfortunately the DoVi Profile 5 only works for direct playing and the transcoding will produce incorrect color. HDR10, HDR10+ and HLG are verified to work.

These are leftovers to be removed as the tone mapping option is separated to make the front-end clients' config easier.

Reply:
We already support remuxing DoVi P5. While tonemapping and transcoding it is only available in libplacebo and our custom CUDA and OpenCL filters. I don't think Apple will enable DoVi support in videotoolbox filters because for commercial products it usually requires licensing from Dolby. Also, HDR10 and HLG to SDR are easy because they use static metadata.